notebooks/test_alert.ipynb

```diff
@@ -0,0 +1,112 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import dash_mdc_neptune as mdc\n",
+    "\n",
+    "from dash import dcc, Input, Output\n",
+    "import jupyter_dash"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "input_text = dcc.Input(id='input_text')\n",
+    "\n",
+    "box = mdc.Box(children=[input_text])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Dashboard layout\n",
+    "\n",
+    "tab = mdc.Tab(\n",
+    "    children=[box],\n",
+    "    tabs=[\n",
+    "        {\"label\": \"Test alert pop up\"},\n",
+    "    ]\n",
+    ")\n",
+    "section = mdc.Section(\n",
+    "    orientation=\"columns\",\n",
+    "    children=tab,\n",
+    "    cards=[{\"title\": \"Card 1\"}]\n",
+    ")\n",
+    "\n",
+    "page = mdc.Page(orientation=\"columns\", children=section)\n",
+    "navbar = mdc.NavBar(title=\"Custom dash\")\n",
+    "alert = mdc.Alert(id='alert')\n",
```

Comment:
🐼 

Is it required to specify an ID to the component?

Reply:
Yes, otherwise I get the error in the console: `Attempting to connect a callback Output item to component alert but no component with that id exist in the layout`

Reply:
@zyassine but isn't the ID set by default (which is already the case on all other components)? That would indicate the optional value isn't set by default in the Dash Python class.

Reply:
@pieeri you're right. I think we only see this error now because we use 'alert' id in a callback